include/HumanView.h

```diff
@@ -25,7 +24,7 @@ class HumanView : public View {
         sf::Keyboard::Key shoot;
     
     public: 
-        HumanView(sf::RenderWindow& App, GameLogic& logic): View( logic ), display(App) {};
+        HumanView(sf::RenderWindow& App, GameLogic& logic, sf::View& view): View( logic ), display(App), view(view) {};
```

Comment:
Can you rename the parameter view to something other than "view" so that there is more clarity when you write "view(view)"